index.html

```diff
@@ -6,6 +6,22 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="/favicon.ico" />
     <title>Cuttle</title>
+
+    
+    <meta property="og:url" content="https://www.cuttle.cards/" />
+    <meta property="og:title" content="Cuttle Cards Online" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="/public/img/cuttle_meta_img.jpg/" />
+    <meta property="og:description"
+        content="Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free!" />
+
+    <meta name="twitter:site" content="https://unacog.com/">
+    <meta name="twitter:title" content="Unacog About">
```

Comment:
What's this one for?

Reply:
This was a copy paste mistake! It should be chagned to cuttle.card with the https

Reply:
Gotcha. Then I think we just want 
```suggestion
    <meta name="twitter:title" content="Cuttle">
```